### PR TITLE
Post IETF 119 Updates

### PR DESCRIPTION
**Pull request by OR13**
- Request registration of receipt parameter
- Add experimental range to reduce provisional registration requests
- editorial cleanup

### CODE REVIEW

File: draft-ietf-cose-merkle-tree-proofs.md [hunk 1]
```diff
@@ -123,38 +133,37 @@ Entry:
...
 
-* Name: The name of the verifiable data structure
-* Value: The identifier for the verifiable data structure
-* Description: The identifier for the verifiable data structure
-* Reference: Where the verifiable data structure is defined
-
 | Name            | Value | Description                      | Reference
 |---
 | N/A             | 0     | N/A                              | N/A
 | RFC9162_SHA256  | 1     | SHA256 Binary Merkle Tree        | {{-certificate-transparency-v2}}
```
Comment (robinbryce) [inline draft-ietf-cose-merkle-tree-proofs.md:159]: Whats the deal with the experimental slots ?

Comment (OR13) [reply draft-ietf-cose-merkle-tree-proofs.md:160]: https://github.com/cose-wg/draft-ietf-cose-merkle-tree-proofs/pull/18/files#diff-e456ca8fff23262339949e1ef0e3b1f5c757b63b05b691c26d656654bef76d6fR148

^ if this is not clear, please suggest changes to it

File: draft-ietf-cose-merkle-tree-proofs.md [hunk 2]
```diff
@@ -83,35 +88,41 @@ Editorial Note (To be removed by RFC Editor).
...
   This property is conceptually similar to COSE Header Parameter "epk" (-1) or CBOR Web Token (CWT) claim "cnf" (8), it is applied to a verifiable data structure, to confirm a property.
-  For example an encrypted message might be decrypted using epk and a private key, a digital signature for authentication might be verified using cnf and the (CWT) claim "nonce" and "audience", and an inclusion proof for a binary merkle tree might be verified with vdp and some entry that is being tested or inclusion in the tree.
-
+  For example an encrypted message might be decrypted using epk and a private key, a digital signature for authentication might be verified using cnf and the (CWT) claim "nonce" and "audience", and an inclusion proof for a binary merkle tree might be verified with VDP and some entry that is being tested or inclusion in the tree.
 
 Proof Type:
 
 : A verifiable process, that proves properties of a Verifiable Data Structure.
+  For example, a VDS such as a binary merkle tree, can support multiple proofs of type "inclusion" where each proof, confirms a given entry is included in a merkle root.
```
Comment (henkbirkholz) [inline draft-ietf-cose-merkle-tree-proofs.md:125]:
```suggestion
  For example, a VDS, such as a binary merkle tree, can support multiple proofs of type "inclusion" where each proof confirms that a given entry is included in a merkle root.
```

**Review by henkbirkholz — APPROVED** [review]
refs provide clarity & usage lgtm

**Comment by ad-l** [comment]
Looking good, thanks

**Pull request merged**